ES/ESmy018.xml

```diff
@@ -32,86 +32,185 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                         ለእግዝእትነ፡ <hi rend="rubric">ማርያም፡</hi> ወላዲተ፡ አምላክ፡ ዘኢይክሉ፡ ሰማያውያን፡ ወምራውያን፡ ፈጽሞ፡ ውዳሴሃ፡ ኢሰማያውያን፡ ወኢምድራውያን፡
+                      ጸሎታ፡ ወበረከታ፡ ይሀሎ፡ ምስለ፡ ገብሩ፡ ተክለ፡ እዎስጣቴዎስ፨ ዝውእቱ፡ ቅድመ፡ ተአምር፡ ይትነበብ፡
+                      </incipit>
+                      </msItem>
+
+                      <msItem xml:id="ms_i1.2">
+                       <locus from="3ra" to="4va"/>
+                       <title type="complete" ref="LIT3584Intro"/>
+                     <note>Miracles</note>
                   </msItem>
-                  <!--The data in the items below refers to other items. please remove this items and move the information to the appropriate place.-->
-                  <msItem xml:id="ms_i2">
+                  <msItem xml:id="ms_i1.">
+                   <locus from="3ra" to="4va"/>
+                   <title type="complete" ref="LIT3584Intro"/>
+                 <note>Miracles</note>
+              </msItem>
+
+                  <msItem xml:id="ms_i1.30">
                      <note>The last miracle (<locus target="#43ra">43ra-</locus>va) is added by a
                         different, crude hand. The incipit of Text IV is not rubricated.</note>
                   </msItem>
+
                   <colophon xml:id="coloph1">
                      <locus target="#34va">34va</locus>, l.8: ዛቲ፡ መጽሐፍ፡ ዘአጽሐፋ፡ ገብረ፡ ክርስቶስ፡ ወልደ፡ ቀሲስ፡
                      ስምዖን፡ ወጸሐፊሃኒ፡ ፍቅረ፡ ኢየሱስ፡ ቀሲስ፡ ዘደንጐሎ። ወስዩመ፡ ደብረ፡ ማዕፆ፡ ተክለ፡ ኤዎስጣቴዎስ፡ ወልደ፡ ጐቢስ፡
                      ትዕቀበነ፡ ኅ[<locus target="#34vb">34vb</locus>, l.8]ቡረ፡ እግዚእትነ፡ ማርያም፡ በጸሎታ፡ ድንግል፡
                      ዛቲ፡ ርግበ፡ ኤፍራታ፡ ኃባኒተ፡ ነድ፡ በክልኤ፡ ኣጥባታ፡ ከመ፡ ሐገፋ፡ ወሥሙር፡ ወልታ፡ {ለገብርኪ፡ ዘባነ፡ ክርስቶስ፡
                      ወለኵልነ፡ ሕዝበ፡ ክርስትያን፡ ዓሜን።} The part of the colophon mentioning Zabana Krǝstos
-                     was added later. </colophon>
-                  <colophon xml:id="coloph2"> f.74ra: ተፈጸመ፡ በዝየ፡ መጽሐፈ፡ መላእክት፡ ዘቅዱስ፡ ዮሐንስ፡ መጥምቀ፡
-                     መለኮት፡  ወጸሐፊሁኒ፡ ፍቁረ፡ ኢየሱስ፡ ዘቅፍርያ፡ ዘአህጒረ፡ ደንጐሎ። ወአጽሐፊሁኒ፡ ገብረ፡ ክርስቶስ፡ ወልደ፡ ቀሲስ፡
-                     ስምዖን፡ ዘደብረ፡ ማዕፆ፡ ስረዩ፡ ሊተ፡ ወባርኩነ፡ ለኃጥኣን፡ በጊዜ፡ ዕጣን፡ ወቍርባን። [fol.74ra] ማኅፀንትክሙ፡
-                     ማኅፀንትክሙ፡ ኢትርስዑነ<locus target="#74">74</locus>: Herewith the book of the
-                     epistles of St John, the Baptist of the Divinity, is completed. And the scribe
-                     is Fǝqurä ʾIyasus of Qǝfrǝya, of the countries of Dängʷälo. And the
-                     commissioner is Gabra Krǝstos, a son of priest Sǝmʿon of Dabra Maʿḍo. Forgive
-                     me and bless us, the sinful, in the time of incense and communion. [fol.74ra]
-                     Your intercession, your intercession do not forget for us! The part of the
-                     colophon mentioning Zabana Krǝstos was added later. </colophon>
+                     was added later.
+                   </colophon>
+
+                    </msItem>
+                     <msItem xml:id="ms_i4">
```

Comment:
is this still part of the Taammera Maryam, or, as in the original summary, a new text? 
in the latter case you need to close the old item and open a new one
```
</msItem>
<msItem xml:id="ms_i2">
```
see the structure in the comment above

Reply:
ok